MediaBrowser.Api/Images/RemoteImageService.cs

```diff
@@ -147,9 +147,8 @@ public async Task<object> Get(GetRemoteImages request)
         {
             var item = _libraryManager.GetItemById(request.Id);
 
-            var images = await _providerManager.GetAvailableRemoteImages(item, new RemoteImageQuery
+            var images = await _providerManager.GetAvailableRemoteImages(item, new RemoteImageQuery(request.ProviderName)
             {
-                ProviderName = request.ProviderName,
```

Comment:
why?

Reply:
Now we guarantee that the property has an initial value